### ISSUE TITLE
[FEATURE] Multi-select?

### ISSUE THREAD

**Issue by pm64** [zleao/zoft.MauiExtensions.AutoCompleteEntry#1]
What a beautiful control.. but I wonder if there's any plan to add multi-selection support. Selections would appear in the entry field as "chips" or "tokens" (not sure what the preferred terminology is).

**Comment by zleao** [comment]
Having a multi-selection option would be great, but not sure about using chips/tokens in the text field. It feels that we would need to use a different control for that. I imagine that by chips, you mean something rhay could be removed by clicking on a 'x' next to the text itself, correct?

**Comment by pm64** [comment]
Yes, that's what I'm referring to. How do you picture it working? The "chip" approach seems well-suited for both desktop and mobile.
